example/get-media-crmls.rb

```diff
@@ -0,0 +1,38 @@
+require 'rets'
+
+client = Rets::Client.new({
+  login_url: 'url',
+  username: 'username',
+  password: 'password',
+  version: 'RETS/1.7.2' 
+})
+
+begin
+    client.login
+rescue => e
+    puts 'Error: ' + e.message
+    exit!
+end
+
+puts 'We connected! Lets get all the photos for a property...'
```

Comment:
The code above here is already documented here: https://github.com/estately/rets/blob/master/example/connect.rb I think we can skip it. You could add a comment referencing that document.